packages/twenty-server/src/engine/core-modules/billing-webhook/utils/transform-stripe-price-event-to-database-price.util.ts

```diff
@@ -4,53 +4,40 @@ import type Stripe from 'stripe';
 
 import { BillingPriceBillingScheme } from 'src/engine/core-modules/billing/enums/billing-price-billing-scheme.enum';
 import { BillingPriceTaxBehavior } from 'src/engine/core-modules/billing/enums/billing-price-tax-behavior.enum';
-import { BillingPriceTiersMode } from 'src/engine/core-modules/billing/enums/billing-price-tiers-mode.enum';
 import { BillingPriceType } from 'src/engine/core-modules/billing/enums/billing-price-type.enum';
 import { SubscriptionInterval } from 'src/engine/core-modules/billing/enums/billing-subscription-interval.enum';
 import { BillingUsageType } from 'src/engine/core-modules/billing/enums/billing-usage-type.enum';
 
 export const transformStripePriceEventToDatabasePrice = (
-  data: Stripe.PriceCreatedEvent.Data | Stripe.PriceUpdatedEvent.Data,
+  data: Stripe.Price,
 ) => {
   return {
-    stripePriceId: data.object.id,
-    active: data.object.active,
-    stripeProductId: String(data.object.product),
-    stripeMeterId: data.object.recurring?.meter,
-    currency: data.object.currency.toUpperCase(),
-    nickname: data.object.nickname === null ? undefined : data.object.nickname,
-    taxBehavior: data.object.tax_behavior
-      ? getTaxBehavior(data.object.tax_behavior)
+    stripePriceId: data.id,
```

Comment:
function / file naming is not right anymore then!

Why this change?

Reply:
I don't get the issue. WDYM by "not right"?